runtime/calamari/src/fee.rs

```diff
@@ -128,9 +132,22 @@ mod multiplier_tests {
 			.max_total
 			.unwrap() - 10;
 
-		let base_fee = <Runtime as pallet_transaction_payment::Config>::WeightToFee::calc(
-			&frame_support::weights::constants::ExtrinsicBaseWeight::get(),
-		);
+		// remark extrinsic is chosen arbitrarily for benchmark as a small, constant size TX.
+		let remark = Call::System(frame_system::Call::<Runtime>::remark_with_event {
+			remark: vec![1, 2, 3],
+		});
+		let len: u32 = remark.clone().encode().len() as u32;
+		let remark_weight: Weight =
+			<Runtime as frame_system::Config>::SystemWeightInfo::remark(len);
+		let max_number_of_remarks_per_block = (block_weight / remark_weight) as u128;
+		let per_byte = <Runtime as pallet_transaction_payment::Config>::TransactionByteFee::get();
+		// length fee. this is not adjusted.
+		let len_fee = max_number_of_remarks_per_block * per_byte.saturating_mul(len as u128);
```

Comment:
why the first op use `*` and the second use `saturating_mul`?

Reply:
Just inconsistency, corrected it.